AZ3166/src/cores/arduino/httpserver/app_httpd.cpp

```diff
@@ -161,21 +161,35 @@ int web_send_wifisetting_page(httpd_request_t *req)
         break;
       }
     }
-    
+
     if (shouldSkip)
     {
       continue;
     }
-    
-    ssidLen = strlen((char *)wifiScanResult[i].get_ssid());
+
+    ssid = (char *)wifiScanResult[i].get_ssid();
+    ssidLen = strlen(ssid);
+    if (ssidLen == 15 && strncmp(ssid, "AZ-", 3) == 0) {
```

Comment:
>15 [](start = 19, length = 2)

Use a constant to represent the fix size of DevKit AP SSID is better than hard-code here.